EventFilter/EcalTBRawToDigi/src/DCCSRPBlock.h

```diff
@@ -36,7 +36,7 @@ class DCCTBSRPBlock : public DCCTBBlockPrototype {
 	protected :
 		
 		void dataCheck();
-		
+		using DCCTBBlockPrototype::increment;
 		void  increment(uint32_t numb);
```

Comment:
Since the base class function has a default parameter for the second argument, one could call the base class function instead of the overriding function. They both would take one argument. If you made a call through a base class pointer is would call the base class function.